src/store/plugins/click-on-map-management.plugin.js

```diff
@@ -82,6 +84,12 @@ const clickOnMapManagementPlugin = (store) => {
                         features: newSelectedFeatures,
                         ...dispatcher,
                     })
+                    if (store.getters.noFeatureInfo && newSelectedFeatures.length > 0) {
```

Comment:
On a click I would always force the default behavior, otherwise the behavior will not be consistent !

Reply:
After testing I saw the point here, when selecting a feature, then put the info in bottom panel, then selecting another feature, in this case you still want the feature in bottom panel. So this is ok, but I would add a small comment here so for the code reader it is clearer.